Plugins/DataSource/Barometer/Barometer.cs

```diff
@@ -24,18 +24,16 @@ public abstract class Barometer : SensorBase
         /// <param name="UID">Sensor ID</param>
         public override void Init(IPConnection connection, string UID)
         {
-            if (_Bricklet != null)
+            if (_Bricklet == null)

```

Comment:
To have a common handling, maybe `_Bricklet` should be named also `Bricklet` like in the other `IDataSource` plugins?